docs/fleet/index.md

```diff
@@ -0,0 +1,43 @@
+title: Fleet Overview
+description: Overview of Fleet - Sematext Agent management panel
+
+The Fleet section of Sematext Cloud gives you an overview of all the Sematext Agents that are running in your systems.
+![Fleet screen](../images/fleet/fleet.png)
+
+At first glance, you can see the following data in the table:
+* *Host*: Name of the hosts in which the Sematext Agent is installed. Clicking on the name will open the Agent Information Panel.
+* *Version*: Installed version of the Sematext Agent and whether important updates are available. If they are, you can update the agent by just clicking the link and following the instructions. You can also find those instructions [here](https://sematext.com/docs/monitoring/spm-faq/#agent-updating).
+* *Status*: Indicates whether the Sematext Agent is reporting any errors.
+* *Started*: Time elapsed since the Sematext Agent began sending data to Sematext Cloud.
+* *Infra App*: the Infra App which is receiving the data from the Sematext Agent. Clicking on the app name will navigate you to the overview of the corresponding app in the Infrastructure section of Sematext Cloud.
+
+# Agent Information Panel
+## Info
+The Info panel contains some information about the host system and installed Agents. 
+
+If the Agents are installed in containers, you will see some information regarding Kubernetes, like Namespace and Node. If, however, the Agents are installed in a baremetal environment, this panel will show information related to the Agent process, like the Pid. 
```

Comment:
Agent that installed in container does not necessary have to be part of Kubernetes. 
Agent can be installed in container in several different environments:(Docker, Swarm, Kubernetes, OpenShift, Nomad)
If agent is installed in container in Kubernetes environment then we will show info like Namespace, Node...

So:
"If the Agents are installed in containers," => "If the Agents are installed in Kubernetes environment,"